src/test/avocado-test.ts

```diff
@@ -225,4 +225,17 @@ describe('avocado', () => {
     const r = await avocado.avocado({ cwd: 'src/test/parse_this_folder', env: {} }).toArray()
     assert.strictEqual(r.length, 0)
   })
+
```

Comment:
Add a positive test when existing readme.

Reply:
postive test already exist in previous test